libs/datasets/timeseries.py

```diff
@@ -479,9 +510,14 @@ def from_combined_dataframe(
 
     @staticmethod
     def from_csv(path_or_buf: Union[pathlib.Path, TextIO]) -> "MultiRegionTimeseriesDataset":
-        return MultiRegionTimeseriesDataset.from_combined_dataframe(
+        mrts = MultiRegionTimeseriesDataset.from_combined_dataframe(
             common_df.read_csv(path_or_buf, set_index=False)
         )
+        if isinstance(path_or_buf, pathlib.Path):
+            provenance_path = pathlib.Path(str(path_or_buf).replace(".csv", "-provenance.csv"))
+            if provenance_path.is_file():
```

Comment:
should this be `exists`?

Reply:
Hmmm, I guess that if a non-file exists with that name it is probably better to let pd.from_csv fail on it than silently ignore it. Changed.